### PR TITLE
chore(i18n): Spelling of ID

### PR DESCRIPTION
**Pull request by rakekniven**
Reported at Transifex

### CODE REVIEW

**Review by joshtrichards — APPROVED** [review]

**Comment by susnux** [comment]
> why adding milestone NC31?

Because every PR against master is milestone 31 or later, of course a backport for stable30 is possible, then that backport PR will target milestone 30.0.x

File: lib/private/Share20/Manager.php [hunk 1]
```diff
@@ -762,7 +762,7 @@ public function updateShare(IShare $share) {
 		try {
 			$originalShare = $this->getShareById($share->getFullId());
 		} catch (\UnexpectedValueException $e) {
-			throw new \InvalidArgumentException($this->l->t('Share does not have a full id'));
+			throw new \InvalidArgumentException($this->l->t('Share does not have a full ID'));
```
Comment (susnux) [inline lib/private/Share20/Manager.php:765]: Yet another place that should never been translated, this will never reach users directly but only logs.

**Review by susnux — APPROVED** [review]
makes sense, though should have been never translated

**Comment by susnux** [comment]
@rakekniven was not yet backported to 30, no backport atm

**Pull request merged**